OpenRA.Mods.Common/Traits/OwnerLostAction.cs

```diff
@@ -0,0 +1,52 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2018 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.Linq;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	public enum LostAction { ChangeOwner, Dispose, Kill }
```

Comment:
This is exported into the traits namespace, so can we please rename it to something a bit more descriptive like `OwnerLostActionType`?